### PR TITLE
[rubysrc2cpg] Unit Tests for Identifier Nodes

### PR DESCRIPTION
**Pull request by pandurangpatil**
1. `identifier` to `method parameter in` traversal
2. `identifier` to `local` traversal
3. across the file additional unit test

### CODE REVIEW

File: joern-cli/frontends/rubysrc2cpg/src/test/scala/io/joern/rubysrc2cpg/passes/ast/IdentifierLocalTests.scala [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+      |""".stripMargin)
+
+  // TODO: Need to be fixed.
+  "be correct for local x in method1" ignore {
+    val List(method) = cpg.method.nameExact("method1").l
+    method.block.ast.isIdentifier.l.size shouldBe 2
+    val List(indentifierX, _) = method.block.ast.isIdentifier.l
+    indentifierX.name shouldBe "x"
+
+    val localX = indentifierX._localViaRefOut.get
```
Comment (DavidBakerEffendi) [inline joern-cli/frontends/rubysrc2cpg/src/test/scala/io/joern/rubysrc2cpg/passes/ast/IdentifierLocalTests.scala:42]: One can also go via the `Local` node via `referringIdentifiers`

Comment (pandurangpatil) [reply joern-cli/frontends/rubysrc2cpg/src/test/scala/io/joern/rubysrc2cpg/passes/ast/IdentifierLocalTests.scala:42]: Do you mean to add a unit test for traversal from the Local node to the identifier?

Comment (DavidBakerEffendi) [reply joern-cli/frontends/rubysrc2cpg/src/test/scala/io/joern/rubysrc2cpg/passes/ast/IdentifierLocalTests.scala:42]: Not necessarily a unit test, but depending on how you query it, the intended API is from `LOCAL -> IDENTIFIER` using `referringIdentifiers` or from `IDENTIFIER -> LOCAL` via `refsTo` which is perhaps less explicit about the node it returns.

Comment (pandurangpatil) [reply joern-cli/frontends/rubysrc2cpg/src/test/scala/io/joern/rubysrc2cpg/passes/ast/IdentifierLocalTests.scala:42]: Added `refsTo` check for one of the unit test and pushed the respective test. `refsTo` also not working.

**Review by DavidBakerEffendi — APPROVED** [review]

**Pull request merged**